pyo3-polars/src/lib.rs

```diff
@@ -234,6 +512,100 @@ impl IntoPy<PyObject> for PyDataFrame {
     }
 }
 
+impl IntoPy<PyObject> for PyAnyValue<'_> {
+    fn into_py(self, py: Python) -> PyObject {
+        let utils = UTILS.as_ref(py);
+        match self.0 {
+            AnyValue::UInt8(v) => v.into_py(py),
+            AnyValue::UInt16(v) => v.into_py(py),
+            AnyValue::UInt32(v) => v.into_py(py),
+            AnyValue::UInt64(v) => v.into_py(py),
+            AnyValue::Int8(v) => v.into_py(py),
+            AnyValue::Int16(v) => v.into_py(py),
+            AnyValue::Int32(v) => v.into_py(py),
+            AnyValue::Int64(v) => v.into_py(py),
+            AnyValue::Float32(v) => v.into_py(py),
+            AnyValue::Float64(v) => v.into_py(py),
+            AnyValue::Null => py.None(),
+            AnyValue::Boolean(v) => v.into_py(py),
+            AnyValue::String(v) => v.into_py(py),
+            AnyValue::StringOwned(v) => v.into_py(py),
+            AnyValue::Categorical(idx, rev, arr) | AnyValue::Enum(idx, rev, arr) => {
+                let s = if arr.is_null() {
+                    rev.get(idx)
+                } else {
+                    unsafe { arr.deref_unchecked().value(idx as usize) }
+                };
+                s.into_py(py)
+            }
+            AnyValue::Date(v) => {
+                let convert = utils.getattr(intern!(py, "_to_python_date")).unwrap();
+                convert.call1((v,)).unwrap().into_py(py)
+            }
+            AnyValue::Datetime(v, time_unit, time_zone) => {
+                let convert = utils.getattr(intern!(py, "_to_python_datetime")).unwrap();
+                let time_unit = time_unit.to_ascii();
+                convert
+                    .call1((v, time_unit, time_zone.as_ref().map(|s| s.as_str())))
+                    .unwrap()
+                    .into_py(py)
+            }
+            AnyValue::Duration(v, time_unit) => {
+                let convert = utils.getattr(intern!(py, "_to_python_timedelta")).unwrap();
+                let time_unit = time_unit.to_ascii();
+                convert.call1((v, time_unit)).unwrap().into_py(py)
+            }
+            AnyValue::Time(v) => {
+                let convert = utils.getattr(intern!(py, "_to_python_time")).unwrap();
+                convert.call1((v,)).unwrap().into_py(py)
+            }
+            // TODO: need help here
+            AnyValue::Array(_v, _) | AnyValue::List(_v) => {
+                todo!();
+                // PySeries(v).to_list()
```

Comment:
Help needed :)

Implementing the `to_list` function for the `PySeries` would add a lot more code because all of this would need to be copied: https://github.com/pola-rs/polars/blob/main/py-polars/src/series/export.rs#L16 including the implementation and all the traits for the `DataType`.

Am i correct in assuming this?
I'm feeling confident that I could "implement" all of this but I didn't want to get started before knowing if I'm actually correct